tests/test_snapshot_download.py

```diff
@@ -357,3 +359,8 @@ def test_download_model_with_ignore_regex(self):
 
     def test_download_model_with_ignore_regex_list(self):
         self.check_download_model_with_regex(["*.git*", "*.pt"], allow=False)
+
+
+def test_snapshot_download_import():
+    with pytest.warns(FutureWarning, match="has been made private"):
+        from huggingface_hub.snapshot_download import snapshot_download  # noqa
```

Comment:
Could we also check that the method is the same as the one imported from `from huggingface_hub import snapshot_download`?

Reply:
Done